cmd/ddev/cmd/commands.go

```diff
@@ -266,18 +298,23 @@ func makeHostCmd(app *ddevapp.DdevApp, fullPath, name string) func(*cobra.Comman
 	}
 
 	return func(cmd *cobra.Command, cobraArgs []string) {
-		status, _ := app.SiteStatus()
-
-		app.DockerEnv()
+		if app != nil {
+			status, _ := app.SiteStatus()
+			app.DockerEnv()
+			_ = os.Setenv("DDEV_PROJECT_STATUS", status)
+		} else {
+			_ = os.Setenv("DDEV_PROJECT_STATUS", "")
+		}
```

Comment:
Not sure if we need this or not? I figured it's safer to have it and not need it, rather than not have it and find out later we do.